XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1376,6 +1349,31 @@
             "VariableSizedWrapGrid",
             "RelativePanel"
           ]
+        },
+                {
+          "UniqueId": "RadioButtons",
+          "Title": "RadioButtons",
+          "Subtitle": "A control that displays a group of related RadioButton controls with keyboarding and accessibility support.",
+          "ImagePath": "ms-appx:///Assets/RadioButtons.png",
+          "Description": "A control that displays a group of related RadioButton controls with keyboarding and accessibility support.",
```

Comment:
Should we also mention that it allows passing simple text or other UI elements and it automagically wraps those? This wording seems a bit like it doesn't add much to existing RadioButton_s.

Reply:
Discussed here: https://github.com/microsoft/Xaml-Controls-Gallery/pull/500#discussion_r453809040